stdlib/public/core/Integers.swift.gyb

```diff
@@ -2104,23 +2107,21 @@ ${overflowOperationComment(x.operator)}
   ///     // z == 2000
   ///
   /// - Parameter other: The value to multiply this value by.
-  /// - Returns: A tuple containing the high and low parts of the result of
-  ///   multiplying this value and `other`.
+  /// - Returns: A tuple containing the high and low parts of the product of
+  ///   this value and `other`.
   func multipliedFullWidth(by other: Self) -> (high: Self, low: Self.Magnitude)
-  // FIXME(integers): figure out how to return DoubleWidth<Self>
 
-  /// Returns a tuple containing the quotient and remainder of dividing the
-  /// given value by this value.
+  /// Returns a tuple containing the quotient and remainder obtained by dividing
+  /// the given value by this value.
   ///
   /// The resulting quotient must be representable within the bounds of the
   /// type. If the quotient of dividing `dividend` by this value is too large
   /// to represent in the type, a runtime error may occur.
   ///
   /// - Parameter dividend: A tuple containing the high and low parts of a
-  ///   double-width integer. The `high` component of the value carries the
-  ///   sign, if the type is signed.
```

Comment:
This is a confusing remark not found anywhere else, even though a similar tuple is accepted as an argument or returned in many other places.

The two's complement value will differ from its magnitude in _both_ the high and low parts, and it's highly doubtful that anyone will wonder if it's the low component that carries the actual sign bit itself.